libexec/setters/set-ilks-line

```diff
@@ -43,4 +43,3 @@ for token in $tokens; do
         setLine "${token}-${ilk}" "$(jq -r ".tokens.${token}.ilks.${ilk} | .line" "$CONFIG_FILE")"
     done
 done
```

Comment:
Dont we need to keep this newline?

Reply:
There were 2. When you do not leave a final newline Github will show some red icon in the diff.